src/components/Transcriber.tsx

```diff
@@ -573,9 +710,9 @@ const Transcriber = withData(mapRecordsToProps)((props: IProps) => {
       return;
     }
     if (previous.hasOwnProperty(state)) {
-      if (state === ActivityStates.Synced || state === ActivityStates.Done) {
-        await reopenSynced();
-      }
+      //if (state === ActivityStates.Synced || state === ActivityStates.Done) {
+      //  await reopenSynced();
+      //}
```

Comment:
Shall we delete these lines?

Reply:
I put in an issue for this TT-1894 but I should probably put it back in until Dale says I can take it out.